tokio/src/sync/rwlock.rs

```diff
@@ -485,6 +485,26 @@ impl<T: ?Sized> RwLock<T> {
         }
     }
 
+    /// Creates a new instance of an `RwLock<T>` which is unlocked.
+    ///
+    /// # Examples
+    ///
+    /// ```
+    /// use tokio::sync::RwLock;
+    ///
+    /// static LOCK: RwLock<i32> = RwLock::const_new(5);
+    /// ```
+    #[cfg(all(feature = "parking_lot", not(all(loom, test))))]
+    pub const fn const_new(value: T) -> RwLock<T>
```

Comment:
The public API additions need the following attribute to have the `parking_lot` requirement show up in the documentation.
```suggestion
    #[cfg(all(feature = "parking_lot", not(all(loom, test))))]
    #[cfg_attr(docsrs, doc(cfg(feature = "parking_lot")))]
    pub const fn const_new(value: T) -> RwLock<T>
```

Reply:
Probably also in the other two. How do I build docsrs-style documentation locally to see the result?

Reply:
Okay, got it now with `cargo +nightly rustdoc --all-features -- --cfg docsrs` after changing the `deny(broken_intra_doc_links)` in `lib.rs`.

Reply:
Building the documentation is described in `CONTRIBUTING.md`.

Reply:
Oh, I see. I wasn’t too far off then.

Reply:
Unfortunately the approach as described in `CONTRIBUTING.md` fails with the same kind of `broken_intro_doc_links` warning/error I had to work around.

Reply:
Which nightly are you on?

Reply:
version is `99111606f 2020-09-11`

the error is mentioning unresolved links to `crate::fs::read_to_string::read_to_string` in `io/util/async_read_ext.rs`, and to `std::io::error::ErrorKind::WouldBlock` in `net/udp/socket.rs`

Reply:
It appears that newest nightly found some new broken links. Fixed in #2834.